src/lite-yt-embed.css

```diff
@@ -44,14 +44,14 @@ lite-youtube > iframe {
 /* play button */
 lite-youtube > .lty-playbtn {
     display: block;
-    width: 68px;
-    height: 48px;
+    width: 100%;
+    height: 100%;
     position: absolute;
     cursor: pointer;
-    transform: translate3d(-50%, -50%, 0);
-    top: 50%;
-    left: 50%;
     z-index: 1;
+    background-size: 68px 48px;
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
We're making the play button/link to cover the whole area and just sizing the background to look as expected.

Reply:
That's very clever. I like it. :).